client/pages/park-details.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import parseRoute from '../lib/parse-route';
+
+export default class ParkDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      park: '',
+      isLoading: true,
+      path: ''
+    };
+  }
+
+  componentDidMount() {
+    const { parkCode } = this.props;
+    const [ path ] = window.location.hash.split('&');
+    fetch(`/api/parks/parkCode/${parkCode}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          park: data.data[0],
+          isLoading: false,
+          path: path
```

Comment:
Try to fight the urge to jam more stuff into state that you can read from elsewhere. this `path` property does not belong in state.